glide.lock

```diff
@@ -1,14 +1,16 @@
-hash: 8b5b47b6ab86b5b212e21bdc4cbf3f9f0c9a1d2a19e96f914a0d02b9b6277324
-updated: 2019-04-16T11:35:54.154136608-05:00
+hash: e3b8c8563c15adba1678f0744325d936f6fb66d04b1801a3d0ad720ed48d31b9
+updated: 2019-05-15T17:12:02.161994906-07:00
 imports:
 - name: code.cloudfoundry.org/clock
   version: 02e53af36e6c978af692887ed449b74026d76fec
 - name: contrib.go.opencensus.io/exporter/ocagent
-  version: 738c4b15ad40c960255f4369c5f0f4e95fc68341
+  version: bbad33429d46c63e46ef4ef90d34893d69fd81aa
+- name: github.com/alecthomas/units
+  version: 2efee857e7cfd4f3d0138cc3cbb1b4966962b93a
 - name: github.com/Azure/azure-sdk-for-go
   version: 1951233eb944a49aa5f7278cba8e3e32a8c958af
   subpackages:
-  - profiles/latest/resources/mgmt/resources
```

Comment:
This is strange...

Reply:
moved? Or not used anymore? I of couse made sure everything would build and pass unit + e2e. Don't think it would go really far if something was missing...